### ISSUE TITLE
AWS Tags

### ISSUE THREAD

**Issue by leobardoramon** [gosha1128/NYUAMS#588]
Implement tags to indicate:

1. Github branch
2. Upgrade testing

**Comment by gosha1128** [comment]
Yes we need github tags and Amazon tags...they should match closely
